cmd/monaco/deploy/deploy.go

```diff
@@ -288,8 +288,7 @@ func validateAuthenticationWithProjectConfigs(projects []project.Project, enviro
 					}
 
 					switch conf.Type.(type) {
-					case config.ClassicApiType,
-						config.SettingsType:
```

Comment:
The problem is the default case also doesn't allow settings with a token.

Reply:
This is causing problems with `cmd/monaco/integrationtest/v2 TestSkip/with_env_override_and_skip_via_env_var`:  https://github.com/Dynatrace/dynatrace-configuration-as-code/actions/runs/11974548692/job/33385877965?pr=1636#step:4:2002

Reply:
Settings just need to be accepted for both possibilities, as long as there is OAuth or a token.

Reply:
Just saw it :), I created a validation case for `SettingsApiType`